### PR TITLE
Swipe

### PR DESCRIPTION
**Pull request by Danwhy**
Allows swiping to switch tag cards when on mobile
ref #110

### CODE REVIEW

File: cms/settings/base.py [hunk 1]
```diff
@@ -124,7 +124,8 @@
     'ldmw-cms.herokuapp.com',
     'ldmw-cms-staging.herokuapp.com',
     '127.0.0.1',
-    'localhost'
+    'localhost',
+    'f9f316b7.ngrok.io'
```
Comment (samhstn) [inline cms/settings/base.py]: maybe don't want this chappy

Comment (Danwhy) [reply cms/settings/base.py]: Good catch, I missed that

**Comment by samhstn** [comment]
Swipe is working well on iphone 5s

I've just noticed though that the left and right buttons are looking strange. Seems like it's only on iphone safari though.

image1

File: cms/static/js/personalise.js [hunk 2]
```diff
@@ -61,10 +62,51 @@ app.ports.listeners.subscribe(function(res) {
...
+    swipeListeners();
   });
 });
 
 app.ports.selectTag.subscribe(function(tags) {
   localStorage.setItem('selected_tags', JSON.stringify(tags));
   app.ports.updateTags.send(tags);
 });
+
+function swipe(el, callback){
```
Comment (SimonLab) [inline cms/static/js/personalise.js:74]: could we use http://package.elm-lang.org/packages/kennib/elm-swipe/latest/Swipe ? The vanilla version looks good :+1:

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

**Pull request merged**